src/routes/contacts/+page.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
+    import Graph from './Graph.svelte';
+    import Icon from 'svelte-icons-pack/Icon.svelte';
+    import BiMap from "svelte-icons-pack/bi/BiMap"; 
+
+    let windowWidth = 0;
+
+</script>
+
+<svelte:window bind:innerWidth={windowWidth} />
+
+<main class="flex flex-col justify-center my-20 -10 ">
+    <section class="flex flex-col text-white font-raleway text-center mb-4">
+        <h1 class="text-2xl font-bold">&lt; Contacta-nos /&gt;</h1>
+    </section>
+    <div class="md:grid grid-cols-2 mx-10">
+        <form class="justify-self-end min-w-[70%]">
+            <label class="m-1 text-white font-source-code" for="email">// Email</label><br>
+            <input class="rounded-lg p-1 text-primary placeholder-primary" type="email" id="email" placeholder="xxxxxxxxxxxxx@gmail.com"><br>
+            <label class="m-1 text-white font-source-code" for="name">// Nome</label><br>
+            <input class="rounded-lg p-1 text-primary placeholder-primary" type="text" id="name"><br>
+            <label class="m-1 text-white font-source-code" for="subject">// Assunto</label><br>
+            <input class="rounded-lg p-1 text-primary placeholder-primary" type="text" id="subject"><br>
+            <label class="m-1 text-white font-source-code" for="message">// Mensagem</label><br>
+            <textarea class="rounded-lg p-1 text-primary placeholder-primary" id="message"></textarea><br>
+
+            <button class="m-1 px-5 py-1 rounded-lg justify-self-start text-white bg-primary" type="submit">Enviar</button>
+        </form>
+        <div class="flex justify-center w-full md:px-3">
+            <Graph />
+        </div>
+    </div>
+    {#if windowWidth >= 768}
+    <img
+        alt="Feup Buildings Outline"
+            src="/images/feup_buildings_md.svg"
+        class="align-center justify-self-center w-full"
+        style="min-width:400px; height:200px; object-fit: none; object-position: center;"
+
+    />
+    {:else}
+    <img
+        alt="Feup Buildings Outline"
+            src="/images/feup_buildings.svg"
+        class="align-center justify-self-center w-full"
+        style="min-width:400px; min-height:160px; object-fit: none; object-position: center;"
+
+    />
+    {/if}
```

Comment:
- The inline style tags are not necessary and can be applied with tailwind (`object-none`, `object-center`, `min-w`and `min-h`), although with slightly different px values (for example, `h-52` is equal to `height:208px;` which I would say is close enough). If you really need exactly 200px, you can use `h-[200px]`, which is not as clean as the previous tailwind class. Moreover, some classes are not very useful from what I've seen, like the min-widths and min-heights (the latter of which can just be converted to height in this case, I think, as the difference is not noticeable). Again, if you feel like they should be preserved, you can pass an arbitrary value to the `min-w` and `min-h`classes using square brackets like I did above.
- This if/else statement can be replicated using a `<picture>` tag (read more about it [here](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction)). Avoids repeating common parts of the image, like the alt text and almost all classes, and doesn't need to use a variable that stores the width of the window, instead changing the source of the image based on a breakpoint. The tag is supported by [every modern browser](https://caniuse.com/picture) (post-2015), but we can discuss its usage in the meeting later today. For the classes that change depending on the window width, just use the tailwind `md:`breakpoint, which has the exact px value as the `if` condition.